src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -875,6 +881,25 @@ public void clickAddQuestionButton() {
         waitForPageToLoad();
     }
 
+    public void clickTemplateModalButton() {
+        addTemplateModalLoadButton.click();
+    }
+
+    public boolean isTemplateQuestionPanelExpanded(int qnNumber) {
+        // need to wait for the panel to be fully expanded before checking the class
+        waitForTextContainedInElementPresence(By.cssSelector(".panel-collapse.collapse.in"), Integer.toString(qnNumber));
+        return browser.driver.findElement(By.id("addTemplateQuestion-" + qnNumber))
```

Comment:
BTW, here you can also use css selector?
`#addTemplateQuestion-${templateQn.qnNumber} in`